Codemine/Extensions/CGRect+Utilities.swift

```diff
@@ -35,7 +35,7 @@ public extension CGRect {
      
      - returns: a new CGRect with the width and height reversed to those of the current one
      */
-    public func rectByReversingSize() -> CGRect {
+	public var reversingSize: CGRect {
```

Comment:
You're the native speaker @chriscombs, so if you think it should be different let's do it that way. Apple is being classic in not obeying their own rules :D